specification/managednetworkfabric/resource-manager/Microsoft.ManagedNetworkFabric/preview/2022-01-15-privatepreview/L2IsolationDomains.json

```diff
@@ -0,0 +1,727 @@
+                "type": "string"
+              }
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "location"
+        },
+        "x-ms-examples": {
+          "L2IsolationDomains_clearNeighborTable_MaximumSet_Gen": {
+            "$ref": "./examples/L2IsolationDomains_clearNeighborTable_MaximumSet_Gen.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ManagedNetworkFabric/l2IsolationDomains/{l2IsolationDomainName}/getArpEntries": {
+      "post": {
+        "tags": [
+          "L2IsolationDomains"
+        ],
+        "operationId": "L2IsolationDomains_getArpEntries",
+        "description": "Clears IPv6 neighbors for this Isolation Domain.",
+        "summary": "Implements the operation to the underlying resources.",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "in": "path",
+            "name": "l2IsolationDomainName",
+            "description": "Name of the L2IsolationDomain.",
+            "required": true,
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "Accepted",
+            "schema": {
+              "$ref": "./common.json#/definitions/GetARPResponse"
+            },
+            "headers": {
+              "Location": {
+                "description": "The URL to retrieve the status of the asynchronous operation.",
+                "type": "string"
+              }
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "location"
+        },
+        "x-ms-examples": {
+          "L2IsolationDomains_getArpEntries_MaximumSet_Gen": {
+            "$ref": "./examples/L2IsolationDomains_getArpEntries_MaximumSet_Gen.json"
```

Comment:
No new async POST actions that return a body beyond a simple "success"/"failure" message are allowed to be added until a new ARM feature is ready in the March 2023 timeframe without your RP implementing extra protections described here:  https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/async-api-reference.md#operation-result-rules